compilers/concrete-compiler/compiler/lib/Dialect/FHE/Transforms/Optimizer.cpp

```diff
@@ -56,6 +56,19 @@ struct OptimizerPartitionFrontierMaterializationPass
     mlir::func::FuncOp func = this->getOperation();
 
     func.walk([&](mlir::Operation *producer) {
+      mlir::IRRewriter rewriter(producer->getContext());
+
+      // Remove the change_partition op.
```

Comment:
I don't understand the TODO? Where is the parameters in the op?

Reply:
They are currently set in the frontend only, they should be forwarded to the op in a future commit